server/src/models/Room.ts

```diff
@@ -1,13 +1,14 @@
 import { User } from '@models/User';
+import { Task } from '@models/Task';
 
 class Room {
   id: number;
-  task: string;
+  task: Task;
```

Comment:
To mozna zrobic prywatne i zrobic metody setTask oraz getTask na wzor tego co robimy z users